ideas/1XX-nail-swarm-process.md

```diff
@@ -0,0 +1,114 @@
+## Preamble
+
+    Idea: 1XX-nail-swarm-process
+    Title: Nail swarm process
+    Status: Draft
+    Created: 2018-03-30
+    Requires (*optional):
+    Replaces (*optional):
+
+## Summary
+Swarms are the primary structure we use to turn ideas into reality at Status.
+People who are excited about bringing an idea to fruition commit to it and make
+sure it happens.
+
+At an individual level, this ensures individual contributors are enabled to
+achieve autonomy, mastery and purpose, while still ensuring accountability.
+
+At a product team / organizational level, swarms allow us to see what is going
+on and which OKRs are targetted, which are not, and generally track progress at
+a reasonable level of granularity.
+
+However, the process for creating, running and completing swarms is currently
+ad-hoc, lacking in accountability and follow-up, and knowledge of how to do this
+properly isn't evenly spread throughout the organization. This swarm will fix
+this.
+
+## Vision
+
+There are two constraints that govern how this structure is designed and
+cultivated. One is that Status the company wants to ship a product, and the
+other is creating a decentralized autonomous organization. Since we are moving
+towards the latter, the structures we are building must be compatible with this
+latter vision, while still being mindful of the first.
+
+The primary difference between the two is that in the former, we are speaking on
+behalf of users, and people are generally compensated on a salary or salary-like
+basis.
+
+We thus distinguish between 'swarms' (type 1) more generally and 'Status LLC
```

Comment:
Would suggest to come up with a better label for `swarms more generally` here. Maybe "swarms using fully open processes" or something. We have a problem where people are not necessarily aware of what the purpose of swarms is so I think being extra clear could help with that.